STAR/2.6.0c/AlignReads.nf

```diff
@@ -11,7 +11,7 @@ process AlignReads {
     
 
     output:
-    tuple sample_id, file("*.bam" ), file("*Unmapped*"), file("*Log.final.out"), file("*Log.out"), file("*SJ.out.tab")
+    tuple sample_id, file("${sample_id}_Aligned.sortedByCoord.out.bam" ), file("*Unmapped*"), file("*Log.final.out"), file("*Log.out"), file("*SJ.out.tab")
```

Comment:
Heeft de unmapped ook niet een 'sample_id' in de naam?

Reply:
Het bestandsnaam is alleen unmapped.out.{mate1/mate2} afhangelijk van SE/PE.